src/blackjack/select-action.js

```diff
@@ -1,3 +1,21 @@
+import cardValue from 'blackjack/card-value'
+import cardSum from 'blackjack/card-sum'
+import shouldHit from 'blackjack/should-hit'
+import shouldSplit from 'blackjack/card-value'
```

Comment:
looks like you imported from `card-value` not `should-split`.

Since you have `shouldSplit` and `shouldHit` (after fixing the import), you won't need `cardValue` or `cardSum` in this module.